dkc/core/models/folder.py

```diff
@@ -63,12 +63,23 @@ class Meta:
     def is_root(self) -> bool:
         return self.parent is None
 
-    def iter_ancestors(self) -> Iterator[Folder]:
-        """Iterate up to the root of the Folder hierarchy, starting with this Folder."""
-        folder = self
-        while folder is not None:
-            yield folder
-            folder = folder.parent
+    @property
+    def ancestors(self) -> models.query.RawQuerySet:
```

Comment:
```suggestion
    def ancestors(self) -> models.query.RawQuerySet[Folder]:
```

Can you see if this works? Sometimes Python is unhappy with these template types, but this is more accurate.